lib/pure/osproc.nim

```diff
@@ -15,7 +15,7 @@ include "system/inclrtl"
 import
   strutils, os, strtabs, streams, cpuinfo
 
-export quoteShell, quoteShellWindows, quoteShellPosix
+export quoteShell, quoteShellWindows, quoteShellPosix, processID
```

Comment:
No, there should be one way to import this symbol and it is in ``os.nim``.

Reply:
done; (although it's quite odd that we have these 2 overloads split up: `os.processID()` and `osproc.processID(Process)`)